### PR TITLE
Query.country doesn't work

### PR DESCRIPTION
**Pull request by XxXgeoXxX**
### Description (*)
Original issue https://github.com/magento/graphql-ce/issues/798

### Manual testing scenarios (*)
Use the following request body to get country information
```{
  country {
    available_regions {
      code
      id
      name
    }
  }
}```

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/DirectoryGraphQl/Model/Resolver/Country.php [hunk 1]
```diff
@@ -53,6 +54,10 @@ public function resolve(
         array $value = null,
         array $args = null
     ) {
+        if (!isset($args['id'])) {
```
Comment (lenaorobei) [inline app/code/Magento/DirectoryGraphQl/Model/Resolver/Country.php]:
``` suggestion
        if (empty($args['id'])) {
```

**Review by lenaorobei — COMMENTED** [review]
Please see one minor comment.

File: app/code/Magento/DirectoryGraphQl/Model/Resolver/Country.php (hunk 1 above)
Comment (XxXgeoXxX) [reply app/code/Magento/DirectoryGraphQl/Model/Resolver/Country.php]: Done

File: app/code/Magento/DirectoryGraphQl/Model/Resolver/Country.php [hunk 2]
```diff
@@ -53,6 +54,10 @@ public function resolve(
         array $value = null,
         array $args = null
     ) {
+        if (empty($args['id'])) {
+            throw new GraphQlInputException(__('"Country id" should be specified.'));
```
Comment (lenaorobei) [inline app/code/Magento/DirectoryGraphQl/Model/Resolver/Country.php]:
``` suggestion
            throw new GraphQlInputException(__('Country "id" value should be specified'));
```

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5602](https://jira.corp.magento.com/browse/ENGCOM-5602) has been created to process this Pull Request

**Pull request merged**